Integrations/PerceptionPoint/PerceptionPoint.yml

```diff
@@ -0,0 +1,83 @@
+commonfields:
+  id: PerceptionPoint
+  version: -1
+name: PerceptionPoint
+display: PerceptionPoint
+category: Email Gateway
+description: Loads incidents from Perception Point and releases falsely quarantined
+  emails.
+configuration:
+- display: Token to use Perception Point's API
+  name: pp_token
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: No. of API loops
+  name: api_loops
+  defaultvalue: "1"
+  type: 0
+  required: false
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Fetch blocked incidents
+  name: fetch_blocked
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Fetch spam incidents
+  name: fetch_spam
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Fetch malicious incidents
+  name: fetch_malicious
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Number of API loops
+  name: api_loops
+  defaultvalue: "1"
+  type: 0
+  required: false
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+- display: Trust any certificate (insecure)
+  name: insecure
+  defaultvalue: ""
+  type: 8 
+  required: false
+- display: Use system proxy
+  name: proxy
```

Comment:
proxy paremeter is here, but there is no proxy handling in the code.
use `handle_proxy()`